integrated_channels/moodle/exporters/learner_data.py

```diff
@@ -57,6 +57,7 @@ def get_learner_data_records(
             MoodleLearnerDataTransmissionAudit(
```

Comment:
nit: While we're here, maybe add in that comment about using 2 duplicate records, like the other providers?